File: smiex_api/smiex_api/Models/Notificacione.cs
using System;
using System.Collections.Generic;

namespace smiex_api.Models;

public partial class Notificacione
{
    public int IdNotificacion { get; set; }

    public string? Titulo { get; set; }

    public string? Mensaje { get; set; }

    public DateOnly? Fecha { get; set; }

    public int? IdUsersend { get; set; }

    public int? IdUserrecieve { get; set; }

    public virtual Usuari? IdUserrecieveNavigation { get; set; }

    public virtual Usuari? IdUsersendNavigation { get; set; }
}

File: simex_app/app/src/main/java/com/example/simex_app/data/models/Notificacione.kt
package com.example.simex_app.data.models

data class Notificacione(
    val idNotificacion: Int,
    val titulo: String?,
    val mensaje: String?,
    val fecha: String?, // DateOnly? se mapea mejor como String? en Retrofit
    val idUsersend: Int?,
    val idUserrecieve: Int?,

    // Relaciones de navegación (Virtual en C#)
    val idUserrecieveNavigation: Usuari? = null,
    val idUsersendNavigation: Usuari? = null
)